.github/workflows/ci.yml

```
@@ -1,56 +1,76 @@
-      with:
-        registry: ghcr.io
-        username: ${{ github.actor }}
-        password: ${{ secrets.GITHUB_TOKEN }}
-
-    # 7️⃣ Push Docker image
-    - name: Push Docker image
-      run: |
-        docker push ghcr.io/${{ github.repository }}:latest
+      # 1 Checkout repository
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)